src/app-layer-protos.h

```diff
@@ -27,6 +27,11 @@
 
 enum AppProtoEnum {
     ALPROTO_UNKNOWN = 0,
+    /* used by the probing parser when alproto detection fails
+     * permanently for that particular stream */
+    ALPROTO_FAILED = 1,
```

Comment:
should have a comment that any update to the value should also be updated in rust I think

Reply:
right